### PR TITLE
Make call to rems admin pims

### PR DESCRIPTION
**Pull request by avirgulto**
## Describe your changes

Test with PR in rems admin for creating the endpoints. 
To test:
Go through workflow of adding rx to pharmacy, do etasu forms and then in pharmacy verify and mark as picked up. This should update the rems admin mongo document with dispense status as picked up

## Issue ticket number and Jira link
 
https://jira.mitre.org/browse/REMS-735 

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] Ensure the target / base branch for any feature PR is set to `dev` not main (the only exception to this is releases from `dev` and hotfix branches)

## Checklist for conducting a review
- [ ] Review the code changes and make sure they all make sense and are necessary. 
- [ ] Pull the PR branch locally and test by running through workflow and making sure everything works as it is supposed to. 

## Workflow 

Owner of the Pull Request will be responsible for merge after all requirements are met, including approval from at least one reviewer. Additional changes made after a review will dismiss any approvals and require re-review of the additional updates. Auto merging can be enabled below if additional changes are likely not to be needed. The bot will auto assign reviewers to your Pull Request for you.

### CODE REVIEW

File: backend/src/routes/doctorOrders.js [hunk 1]
```diff
@@ -177,6 +177,14 @@ router.patch('/api/updateRx/:id/pickedUp', async (req, res) => {
       }
     });
     console.log('Sent RxFill to EHR and received status from EHR', status.data);
+
+    const remsAdminStatus = await axios.post('http://rems-administrator:8090/ncpdp', rxFill, {
+      headers: {
```
Comment (plarocque4) [inline backend/src/routes/doctorOrders.js:182]: Should we expect the return to be xml on this one as well?

File: backend/src/routes/doctorOrders.js [hunk 2]
```diff
@@ -177,6 +177,14 @@ router.patch('/api/updateRx/:id/pickedUp', async (req, res) => {
       }
     });
     console.log('Sent RxFill to EHR and received status from EHR', status.data);
+
+    const remsAdminStatus = await axios.post('http://rems-administrator:8090/ncpdp', rxFill, {
```
Comment (plarocque4) [inline backend/src/routes/doctorOrders.js:181]: make sure to update this endpoint to match the ncpcp/script end point in rems admin.

Comment (plarocque4) [inline backend/src/routes/doctorOrders.js:181]: The rems-administrator endpoint will not work outside of docker. This should be configurable as an environment variable and we can maybe add the capability to forward these messages through the intermediary in the future.

File: backend/src/routes/doctorOrders.js (hunk 1 above)
Comment (plarocque4) [reply backend/src/routes/doctorOrders.js:182]: The xml return is part of the NCPDP SCRIPT RxFill exchange. Even if we do nothing with it we should include the STATUS return to be complete.

File: backend/src/routes/doctorOrders.js [hunk 3]
```diff
@@ -177,6 +177,15 @@ router.patch('/api/updateRx/:id/pickedUp', async (req, res) => {
       }
     });
     console.log('Sent RxFill to EHR and received status from EHR', status.data);
+
+    const remsAdminStatus = await axios.post('http://rems-administrator:8090/ncpdp/script', rxFill, {
```
Comment (plarocque4) [inline backend/src/routes/doctorOrders.js:181]: I see my other comment was marked as outdated, but rems-administrator will only work on docker. That is why I am unable to successfully test this. Can you please make the host/ip configurable?

File: backend/src/routes/doctorOrders.js (hunk 1 above)
Comment (avirgulto) [reply backend/src/routes/doctorOrders.js:182]: oh okay makes sense

**Review by plarocque4 — APPROVED** [review]
looks good

**Pull request merged**